crypto/sshutil/sshutil.go

```diff
@@ -182,49 +171,3 @@ func parseRSA(in []byte) (*rsa.PublicKey, error) {
 	key.N = w.N
 	return &key, nil
 }
-
-// parseECDSA parses an ECDSA key according to RFC 5656, section 3.1.
```

Comment:
What I'm unsure about with this PR is whether we have some reason to prefer these `parse` functions over what is already offered by `crypto/ssh`? These appear to only have been used in `PublicKey` above, which was only ever used to determine the key type in `inspectPublicKey`. I don't see any references to them anywhere else.

Reply:
We're not using that method anymore, but it can be useful if we want to extract information about the key of a certificate, or just extract it the from an SSH certificate, we have commands to do that in an x509 certificate, but not yet for a SSH certificate and we should.

x/crypto/ssh does not provide a way to convert a ssh.PublicKey to the actual key, this is why I created those methods. So we should keep them.